docs: update ceph-teardown doc (#8017)
rook/rook: Documentation/ceph-teardown.md

kubectl delete storageclass csi-cephfs
```

After those block and file resources have been cleaned up, you can then delete your Rook cluster. This is important to delete **before removing the Rook operator and agent or else resources may not be cleaned up properly**.

> ## Automatic Cleanup

Reviewer: How about if we just keep this in the same section? We're just recommending to use the automatic cleanup by default now.
```suggestion
## Delete the CephCluster CR
```

> Edit the cephCluster and add the `cleanupPolicy`

Reviewer 2: ```suggestion
Edit the `CephCluster` and add the `cleanupPolicy`
```

> WARNING: DATA WILL BE PERMANENTLY DELETED AFTER DELETING THE `cephCluster` CRD WITH `cleanupPolicy`.

Reviewer 2: ```suggestion
WARNING: DATA WILL BE PERMANENTLY DELETED AFTER DELETING THE `CephCluster` CR WITH `cleanupPolicy`.
```

```console
kubectl -n rook-ceph patch cephcluster rook-ceph --type merge -p '{"spec":{"cleanupPolicy":{"confirmation":"yes-really-destroy-data"}}}'
```

> Once the cleanup policy is enabled, any new configuration changes in the cephCluster will be blocked. Nothing will happen unitl the deletion of the CR is requested, so this `cleanupPolicy` change can still be reverted.

Reviewer 2: ```suggestion
Once the cleanup policy is enabled, any new configuration changes in the CephCluster will be blocked. Nothing will happen unitl the deletion of the CR is requested, so this `cleanupPolicy` change can still be reverted.
```

> Once the cleanup policy is enabled, any new configuration changes in the CephCluster will be blocked. Nothing will happen unitl the deletion of the CR is requested, so this `cleanupPolicy` change can still be reverted.

Reviewer: ```suggestion
Once the cleanup policy is enabled, any new configuration changes in the CephCluster will be blocked. Nothing will happen until the deletion of the CR is requested, so this `cleanupPolicy` change can still be reverted if needed.
```

> Checkout more details about the `cleanupPolicy` [here](https://rook.io/docs/rook/master/ceph-cluster-crd.html#cleanup-policy)

Reviewer: Actually just refer to the .md files in the links, and the build process replaces with the correct html and branch link
```suggestion
Checkout more details about the `cleanupPolicy` [here](ceph-cluster-crd.md#cleanup-policy)
```

> ## Delete the CephCluster CRD

Reviewer: no need for this header if the other is renamed above

kubectl -n rook-ceph get cephcluster
```

If the `cleanupPolicy` was applied, then wait for the `rook-ceph-cleanup` jobs to be completed on all the nodes.
These jobs will perform the following operations:
- Delete the directory `/var/lib/rook` (or the path specified by the `dataDirHostPath`) on all the nodes
- Wipe the data on the drives on all the nodes where OSDs were running in this cluster

> Note: The cleanup jobs might not start if the resources created on top of Rook Cluster are not deleted completely. [See](ceph-teardown.md#delete-the-block-and-file-artifacts)

Reviewer: nit: add a newline

kubectl -n rook-ceph get cephcluster
```

If the `cleanupPolicy` was applied, then wait for the `rook-ceph-cleanup` jobs to be completed on all the nodes.

> These jobs :

Reviewer: ```suggestion
These jobs will perform the following operations:
```

> - Clean up the `/var/lib/rook` (or the path specified by the `dataDirHostPath`) on all the nodes

Reviewer: ```suggestion
- Delete the directory `/var/lib/rook` (or the path specified by the `dataDirHostPath`) on all the nodes
```

> - Delete the data on the drives on all the nodes

Reviewer: ```suggestion
- Wipe the data on the drives on all the nodes where OSDs were running in this cluster
```

> Note: The cleanup jobs might not start if the resources created on top of Rook Cluster are not deleted completely. [See](https://github.com/rook/rook/blob/master/Documentation/ceph-teardown.md#delete-the-block-and-file-artifacts)

Reviewer: ```suggestion
Note: The cleanup jobs might not start if the resources created on top of Rook Cluster are not deleted completely. [See](ceph-teardown.md#delete-the-block-and-file-artifacts)

```

kubectl delete -f crds.yaml
```

> If the `cleanupPolicy` was applied and the cleanup jobs have completed on all the nodes, then the cluster tear down has been successful. If you have skipped adding the `cleanupPolicy` then follow the manual steps mentioned below to teardown the cluster.

Reviewer: ```suggestion
If the `cleanupPolicy` was applied and the cleanup jobs have completed on all the nodes, then the cluster tear down has been successful. If you skipped adding the `cleanupPolicy` then follow the manual steps mentioned below to tear down the cluster.
```
